### ISSUE TITLE
failing to find tests

### ISSUE THREAD

**Issue by Jaymon** [Jaymon/pyt#26]
given a folder structure like:

```
foo_tests/
    __init__.py
    bar/
        __init__.py
        che_test.py
```

these are failing:

    $ pyt foo.bar
    $ pyt foo.bar.che

I think even though `bar` doesn't have `_test` suffix pyt should find tests if it contains tests inside of `bar`, and I have no idea why `foo.bar.che` is failing, I need to look into that.

Interestingly, this worked:

    $ pyt foo_test.bar.che_test

as did the more specific:

     $ pyt foo_test.bar.che_test.Class.method

So the problem seems to be in the module pathfinding.

**Comment by Jaymon** [comment]
I think the algorithm for something like `foo.bar` would be:

1. check for the existence of `foo_test` if that exists follow it
2. if `foo_test` does not exist then check for `foo`
3. if neither exist then move to the non-module check

**Comment by Jaymon** [comment]
This was closed in b47838f, but I accidentally put #25 in the commit message
